frontend/js/behaviors/core/gridboard.js

```diff
@@ -82,10 +82,21 @@ const gridboard = function(container) {
             
             // Position blocks in the current row
             forEach(row, function(colIndex, block) {
-                // Set the `top` and `left` positions for the block
+                // Set initial state
+                block.classList.remove('s-positioned');
+                block.classList.add('s-animating');
+                
+                // Calculate the final position
                 let leftPosition = colIndex * (colWidth + marginLeft);
                 
+                // Set the position without transition first
                 block.style.position = "absolute";
+                block.style.transition = "none";
+                
+                // Force a reflow to ensure the initial position is set before animation starts
+                void block.offsetWidth;
+                
+                block.style.transition = "all 1s cubic-bezier(0.25, 0.1, 0.25, 1)";
```

Comment:
Minor suggestion: It might be more clear to use milliseconds here so the value on line 111 matches, for example:
```javascript
block.style.transition = "all 1000ms cubic-bezier(0.25, 0.1, 0.25, 1)";
```